math.go

```diff
@@ -378,6 +379,78 @@ var (
 			}
 			return valueNode(nil, "avg", Null, nil), nil
 		},
+		"b64decode_no_padding": func(node *Node) (result *Node, err error) {
```

Comment:
Naming functions like `b64decode_no_padding` and `b64decode_std_padding` makes it quite hard to use.
Imagine that you'll need to select something with the decoded request, like:
```
$..[?(b64decode_no_padding(@.foo) == "bar" || b64decode_std_padding(@.foo) = "bar")]
```
looks a bit overloaded. For the `decoded` function better to combine both `_no_padding` and `_std_padding` together in one `b64decode` function, so it will be used easier:
```
$..[?(b64decode(@.foo) == "bar")]
```

Implementation in this case should be like:
```golang
// first for STD encoded strings
result, err = base64.StdEncoding.WithPadding(base64.StdPadding).DecodeString(sourceString)
if err != nil {
	// then for NO_PAD encoded strings, if the first result with error
	result, err = base64.StdEncoding.WithPadding(base64.NoPadding).DecodeString(sourceString)
}
if err != nil {
	return nil, err
}
```

Reply:
Ok :) I will adjust the decode function